pages/deployment/linux.mdx

```diff
@@ -0,0 +1,334 @@
+settings are:
+
+- `--log-level=WARNING`
+- `--query-execution-timeout-sec=600`
+- `--storage-snapshot-interval-sec=300  `
+- `--storage-snapshot-retention-count=3`
+- `--storage-mode=IN_MEMORY_TRANSACTIONAL`
+
+Of course, other
+[settings](../database-management/configuration.mdx#list-of-configuration-flags)
+are as important, so make sure you get familiar with them as well.
+
+To change the configuration settings, you need to open and edit the
+`memgraph.conf` file and restart the Memgraph service. 
+
+The set flags override the default settings in the `memgraph.conf` file. It is
+also possible to provide an additional configuration file by providing a path to
+it via `--file-flag` or `MEMGRAPH_CONFIG` environment variable. The set values
+in that file will override values in the default `memgraph.conf` file. 
+
+## Secure the database
+
+In the community version of Memgraph, you can create a user and its password for
+security. To achieve that, set `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD`
+environment variables. When you set values for those variables, a user will be
+created if it does not exist, and no one except from that user will be able to
+access the database. 
+
+To authenticate, provide the correct username and password. For example, if you
+want to run Memgraph's CLI `mgconsole` on localhost, run the following command:
+
+```bash
+mgconsole --host=127.0.0.1 --port=7687 --username=user --password=pass
+```
+
+
+<Cards>
+<div className="w-[230px]">
+    <Card
+    title="Learn more about users"
+    href="/database-management/authentication-and-authorization/users"
+    />
+</div>
+</Cards>
+
+In the Enterprise version of Memgraph, you can achieve a higher level of
+security with the role-based access control (RBAC) and fine-grained access
+control. Within fine-grained access control Memgraph offers label-based access
+control.
+
+To set it up, first [enable Memgraph
+Enterprise](/database-management/enabling-memgraph-enterprise), and then run
+[the necessary Cypher
+queries](/database-management/authentication-and-authorization/role-based-access-control)
+to set up privileges properly.
+
+You can add an additional layer of security by enabling SSL encryption.
+
+<Cards>
+<div className="w-[300px]">
+    <Card
+    title="Learn more about SSL encryption"
+    href="/database-management/ssl-encryption"
+    />
+</div>
+</Cards>
+
+## Data persistency 
+
+Data persistence is a crucial aspect of running a database. You want to keep
+your data Persistent if anything goes wrong. Memgraph stores different types of
+information in different places. 
+
+Here are the locations of the different types of data that Memgraph uses: 
+
+- Configuration files: `/etc/memgraph`
+- Logs: `/var/log/memgraph`
+- User-related data: `/usr/lib/memgraph`
+- Graph data: `/var/lib/memgraph`
+
+Based on what you want to backup, consider copying any of the listed folders to
+a safe location.
+
+## Backup data
+
+Memgraph uses  `snapshots` and `wal-files`
```

Comment:
On other places in our docs WAL files are used (it's my fault because I wrote wal-files on my PR hehe)